packages/relay/src/relayer.ts

```diff
@@ -168,7 +168,10 @@ function isApiCredentials(credentials: AutotaskRelayerParams | ApiRelayerParams)
 export function isLegacyTx<TransactionLikeType>(
   tx: TransactionLikeType,
 ): tx is TransactionLikeType & { gasPrice: NonNullable<unknown> } {
-  return 'gasPrice' in tx;
+  // Consider that an EIP1559 tx may have `gasPrice` after
+  // https://github.com/OpenZeppelin/defender/pull/2838
+  // that's why the !isEIP1559Tx check
+  return 'gasPrice' in tx && !isEIP1559Tx(tx);
```

Comment:
I know @spalladino will probably call me a coward but would it be possible to add some parenthesis here 👉 👈 ? It took me a couple of brain cycles to parse the operator precedence

`return ('gasPrice' in tx) && !isEIP1559Tx(tx);`

Reply:
I'm fine adding the suggestion, but it gets removed because of the styling 😞 also the linter is flagging it.
Same happens in the backend where we have an equivalent function.